tracer/src/Datadog.Trace/Configuration/GlobalSettings.cs

```diff
@@ -108,7 +109,7 @@ internal static void SetDebugEnabledInternal(bool enabled)
                 DatadogLogging.UseDefaultLevel();
             }
 
-            LibDatadog.Logger.Instance.SetLogLevel(debugEnabled: enabled);
+            Logger.Instance.SetLogLevel(debugEnabled: enabled);
```

Comment:
I'm actually not a big fan of this because I think it's important to keep the context that this is LibDatadogLogger, and there's already multiple types and variables called `Logger` 😅 

I would suggest adding this `using`:

```csharp
using LibDatadogLog = Datadog.Trace.LibDatadog.Logger;
```

And then using this instead:

```suggestion
            LibDatadogLog .Instance.SetLogLevel(debugEnabled: enabled);
```

(or something similar)

Reply:
fair enough!

I can be convinced with keeping the namespace prefix.